gammapy/region/tests/test_circle.py

```diff
@@ -50,3 +52,11 @@ def test_sky_to_pix_to_sky(self):
         assert_allclose(sky.pos.l, sky2.pos.l)
         assert_allclose(sky.pos.b, sky2.pos.b)
         assert_allclose(sky.radius, sky2.radius)
+
+    def test_area(self):
+        pos = SkyCoord.from_name('crab')
```

Comment:
The `SkyCoord.from_name` method makes a query to the internet SESAME service.
This is better avoided in tests.

You could hard-code the Crab position or choose some random other position, no?
```python
>>> from astropy.coordinates import SkyCoord
>>> SkyCoord.from_name('crab')
<SkyCoord (ICRS): (ra, dec) in deg
    (83.633083, 22.0145)>
```